cassandra.go

```diff
@@ -101,21 +101,20 @@ func (driver *Driver) FilenameExtension() string {
 	return "cql"
 }
 
-func (driver *Driver) Migrate(f file.File, pipe chan interface{}) {
-	var err error
+func (driver *Driver) FileTemplate() []byte {
+	return []byte("")
+}
+
+func (driver *Driver) Migrate(f file.File) (err error) {
 	defer func() {
 		if err != nil {
 			// Invert version direction if we couldn't apply the changes for some reason.
 			if errRollback := driver.session.Query("DELETE FROM "+tableName+" WHERE version = ?", f.Version).Exec(); errRollback != nil {
-				pipe <- errRollback
+				err = fmt.Errorf("%s; failed to rollback version: %s", err, errRollback)
```

Comment:
The initial error is lost here

Reply:
@gravis I think ignoring rollback error here is OK, but read carefully, original error is preserved (at least it's string representation, which is completely fine)

Reply:
ho, my bad :)
sorry